SPA/app.js

```diff
@@ -221,6 +238,32 @@ function isRootIdentity(identity){
 }
 
 /*
+* Handles the "authenticated" event for all Lock log-ins.
+*/
+
+function lockAuthenticated(authResult)
+{
```

Comment:
braces should be right after the method `function lockAuthenticated(authResult) {`